app/ide-desktop/lib/dashboard/src/components/AriaComponents/Form/Form.tsx

```diff
@@ -114,6 +116,15 @@ export const Form = React.forwardRef(function Form<
   // There is no way to avoid type casting here
   // eslint-disable-next-line @typescript-eslint/no-explicit-any,no-restricted-syntax,@typescript-eslint/no-unsafe-argument
   const formOnSubmit = innerForm.handleSubmit(formMutation.mutateAsync as any)
+
+  offlineHooks.useOfflineChange(isOffline => {
```

Comment:
note that some dialogs may need to work on the Local Backend - for example, dialogs for creating e.g. a secret (although this is currently not possible)

Reply:
Good catch! Will add an option to disable this behavior